DuckDuckGo/ProgressView.swift

```diff
@@ -41,6 +41,7 @@ class ProgressView: UIView, CAAnimationDelegate {
     override init(frame: CGRect) {
```

Comment:
The other constructor is technically redundant, but I don't want to change too much here just to fix this.